themes/default/data/team/team/aurelien-requiem.toml

```
@@ -0,0 +1,11 @@
+id = "aureq"
```

We use this value to link to the image, so if you change it to `aurelien-requiem`, things will "just work". 😄 

```suggestion
id = "aurelien-requiem"
```